docs/internals/contributing/writing-documentation.txt

```diff
@@ -341,6 +341,39 @@ documentation:
     Five
     ^^^^
 
+* Add headers for functions, classes, and methods in reference documentation:
+
+  - Each function, class, method, or attribute in reference documentation should
+    have a header using the appropriate heading level (typically "~~~~" style).
+  - Use the exact name of the item as the header text.
+  - Place the header directly above the directive documentation.
+  - For better accessibility and navigation, ensure these headers are
+    consistently applied across all reference documentation.
+
+  For example:
+
+  .. code-block:: rst
+
+    patch_cache_control()
+    ~~~~~~~~~~~~~~~~~~~~
```

Comment:
Underline should have the same number of characters as a function name, it's wrong in both examples.
```suggestion
    ~~~~~~~~~~~~~~~~~~~~~
```